code/game/gamemodes/miniantags/hades/hades.dm

```diff
@@ -88,7 +125,12 @@
 			world << pick(creepyasssounds)
 		else
 			lastsinPerson = world.time
-			var/mob/living/carbon/human/sinPerson = pick(player_list)
+			var/mob/living/carbon/human/sinPerson = pick(living_mob_list)
+			var/depth = living_mob_list.len + 1 // just in case

```

Comment:
yes, and player_list & living_mob_list is the intersection, so it'll be a list of only the mobs that are in BOTH lists